pkg/chains/formats/provenance/provenance.go

```diff
@@ -44,8 +46,11 @@ const (
 )
 
 type Provenance struct {
-	builderID string
-	logger    *zap.SugaredLogger
+	builderID        string
+	logger           *zap.SugaredLogger
+	spireEnabled     bool
+	spireSocket      string
+	spireWorkloadAPI *spire.SpireWorkloadApiClient
```

Comment:
if we have the spireWorkloadAPI, do we still need spireSocket in the struct? could this all be done during the initialization of the Provenance object?

Reply:
removed this and initialized